### DISCUSSION TITLE
Create New Portal Without Requiring a Desktop Folder?

### DISCUSSION THREAD

**Discussion by snowfi6916** [Ross-Patterson/Portals-Desktop-Organization#175] [General]
Is there a way to create a portal without having a folder attached to it? I know that similar apps allow you to just have a portal without having a folder tied to it. It's annoying that I need to have the folder always on my desktop, and if I delete it it deletes the portal too.

Thanks.

**Comment by Ross-Patterson** [comment]
No this is not the intention of the application. I know that apps like Fences do this by actively manipulating your actual desktop icons, Portals will not do anything like that.

The folder(s) do not need to be on the desktop, you can hide the folder(s) away somewhere like Documents.

**Reply by mkolassa** [reply]
I've tucked mine into `%USERPROFILE%\Desktop_Portals` and quite like it there.  Keeps it free of the normal cloud-backup softwares since shortcuts are unique to each computer.

Maybe a new default parent folder could be set instead of just using the Desktop?

**Reply by bandaid** [reply]
It's a different approach. I'll have to rearrange my directory to see how I like this, overall it's simple and quick. I'm not sure why you think creating directories for each portal is necessary?

Personally, i have my desktop shortcuts saved to a nas location. I haven't used anything for several years and just decided why not look for an alternative. I just found yours. I previously used a mindlesssheep alternate (got into heated argument with devs for not providing a 'lock' feature and disabling creation of boundaries, twelve years later rocks became smarter than they) and will never use it. NimiPlaces is good and good dev team.

**Comment by rconstantine** [comment]
This should be more prominently noted on the website. I was going to skip using this application because of this until I found this post.

**Reply by Ross-Patterson** [reply]
This is good feedback, I will try to think of how to make this more obvious

**Comment by wstanzl** [comment]
I agree with @mkolassa, I made a folder %USERPROFILE%\Portals and also set this to the default for new portals. It is the logical place to put it, since "Desktop" itself is also there. 
My suggestion: make that the default in the app.

**Comment by fmammadov** [comment]
I wanted to share my first impressions after installing the app today. I really like the app and its flexibility. While I'm still getting used to the folder structure, I was wondering why folders are mandatory for creating portals. Are there technical challenges or potential copyright issues with Stardock behind this decision?
Creating portals/fences without folders feels more natural when organizing files, and the current approach requires some adjustment. As someone who enjoys exploring new software, I may be a niche user, but I think changing the portal logic could help you attract more users from Fences. This seems like a significant limitation since Fences is more user-friendly - you can create fences and organize files immediately after installation, without thinking about file and folder structures (which might be more suited for advanced users).

Thank you and good luck with your awesome product!
